package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.12.0"
   },
   "scripts": {
-    "test": "xo && npm run prepublish && nyc ava",
+    "test": "xo && npm run prepublish && nyc ava --serial",
```

Comment:
Had to add `--serial` because if they run concurrently, they did not complete on my system. Maybe something to do with Phantom?